requirements.in

```diff
@@ -1,9 +1,9 @@
-pip
+pip==9.0.3
```

Comment:
What's the purpose of this?

Reply:
I could put it back as it was, in fact I was not intent to push this change. The lastest pip version 19.0.3 has a bug and every time I update requirements and it upgrades to the latest version I have many problems. I need to rollback.

Reply:
I think it's okay to leave this in, but it probably wants an explanation in a comment. Is there an upstream issue to link to?

Reply:
I have not seen any issue about this.